pkg/scheduler/framework/detector/detect_framework_test.go

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This file helps to detect whether we changed the interface of Framework.
+// This is important for downstream projects who import the Framework to not
```

Comment:
we specifically care about projects under the github.com/kubernetes org, not others.

Reply:
nit: projects under `github.com/kubernetes*` orgs, which is the default search scope of https://cs.k8s.io/.